hack/make/build-rpm

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -e
+
+DEST=$1
+
+# subshell so that we can export PATH without breaking other things
+(
+	source "$(dirname "$BASH_SOURCE")/.integration-daemon-start"
+
+	# TODO consider using frozen images for the dockercore/builder-rpm tags
+		
+	rpmName=docker-engine
+	rpmVersion="${VERSION%%-*}"
+	rpmRelease=1
+
+	# rpmRelease versioning is as follows
+	# Docker 1.7.0:  version=1.7.0, release=1
+	# Docker 1.7.0-rc1: version=1.7.0, release=0.1.rc1
+	# Docker 1.7.0-dev nightly: version=1.7.0, release=0.0.YYYYMMDD.HHMMSS.gitHASH
+	
+	# if we have a "-rc*" suffix, set appropriate release
+	if [[ "$VERSION" == *-rc* ]]; then
+		rcVersion=${VERSION#*=rc}
+		rpmRelease="0.${rcVersion}.rc${rcVersion}"
+	fi
+
+	# if we have a "-dev" suffix or have change in Git, let's make this package version more complex so it works better
+	if [[ "$VERSION" == *-dev ]] || [ -n "$(git status --porcelain)" ]; then
+		gitUnix="$(git log -1 --pretty='%at')"
+		gitDate="$(date --date "@$gitUnix" +'%Y%m%d.%H%M%S')"
+		gitCommit="$(git log -1 --pretty='%h')"
+		gitVersion="${gitDate}.${gitCommit}"
```

Comment:
This should be `git${gitCommit}`, I think (`YYYYMMDD.HHMMSS.gitHASH`), so that it matches the way RPM packagers (and thus users) expect to see the Git metadata labeled (since that's how this release's exact commit can be looked up trivially, so making that obvious is ++ :smile:).